dj/typing.py

```diff
@@ -227,20 +249,49 @@ def __getitem__(self, keys) -> "ColumnType":
 
     @classmethod
     def _validate_type(cls, type_: str):
-        test = TYPE_PATTERN.match(type_)
-        if test is not None:
-            outer = test.group("outer")
-            inner = test.group("inner")
-            if outer not in COMPLEX_TYPES:
-                raise ColumnTypeError(f"{outer} is not a KNOWN complex type.")
-            inners = inner.split(",") if outer == ColumnType.MAP else inner
-            return ColumnType(outer)[inners]
+        nested = parse_type(type_)[0]
+        if isinstance(nested, list) and len(nested) > 1:
+            if nested[0] not in COMPLEX_TYPES:
+                raise ColumnTypeError(f"{nested[0]} is not a KNOWN complex type.")
```

Comment:
nit: I know this error message was already defined but the all caps looks odd

Reply:
@samredai  WHY

Reply:
NOT EXACTLY SURE 😂

Reply:
I REMOVED IT!